.github/workflows/validate.yaml

```diff
@@ -64,6 +64,7 @@ jobs:
         working-directory: ./src/github.com/${{ github.repository }}
         env:
           WOKE_VERSION: v0.4.1
+          OSDK_VERSION: v1.3.0
```

Comment:
We will need to update this often? Or this part is pretty stable?

Reply:
I'd hope it's stable. 1.0.0 had the very same failures before our fix.

Reply:
The only corner case would be current version to ignore some new feature we add from a a future release of OLM. But I guess it would complain.